shared/actions/wallets.js

```diff
@@ -235,6 +235,7 @@ const loadSendAssetChoices = (state: TypedState, action: WalletsGen.LoadSendAsse
   })
 
 const loadDisplayCurrency = (state: TypedState, action: WalletsGen.LoadDisplayCurrencyPayload) =>
+  action.payload.accountID !== Types.noAccountID &&
```

Comment:
we can call this with noAccountID and get a black bar

Reply:
would isValidAccountID be better?

Reply:
unclear, there's just one value so it doesn't seem necessary